internal/app/wwctl/container/exec/root.go

```diff
@@ -25,17 +25,17 @@ var (
 		},
 		FParseErrWhitelist: cobra.FParseErrWhitelist{UnknownFlags: true},
 	}
-	SyncUser bool
-	binds    []string
-	tempDir  string
-	nodeName string
+	SyncUser   bool
+	binds      []string
+	overlayDir string
+	nodeName   string
 )
 
 func init() {
 	baseCmd.AddCommand(child.GetCommand())
 	baseCmd.PersistentFlags().StringArrayVarP(&binds, "bind", "b", []string{}, "Bind a local path into the container (must exist)")
 	baseCmd.PersistentFlags().BoolVar(&SyncUser, "syncuser", false, "Synchronize UIDs/GIDs from host to container")
-	baseCmd.PersistentFlags().StringVar(&tempDir, "tempdir", "", "Use tempdir for constructing the overlay fs (only used if mount points don't exist in container)")
+	baseCmd.PersistentFlags().StringVar(&overlayDir, "overlaydir", "", "Use tempdir for constructing the overlay fs (only used if mount points don't exist in container)")
```

Comment:
I'm concerned that the use of the term "overlay" here is confusing, given that we already use the term for a primary Warewulf feature. It took me a while to remember what's going on here myself.

Reply:
Right, as command line option we can find another name,